configs/olcf/job_script.olcf.bash

```diff
@@ -16,8 +16,8 @@
 cd $PBS_O_WORKDIR
 
 module unload python
-module use /ccs/proj/cli115/pwolfram/modulefiles/all
-module load python/anaconda-2.7-climate
+module use /ccs/proj/cli127/software/modulefiles/all
```

Comment:
@xylar: what is cli127? I don't have access to it.

Reply:
Hmm, it's the only group I have access to.  Maybe our alcc project?

Reply:
I don't have access to cli115.  Is that the E3SM group on olcf?

Reply:
yep, it's the main e3sm project at OLCF. You should have access to it. Could you ask for it? it should be fairly quick: usual form and I think Mark Taylor is the PI.

Reply:
Do I really need to fill out the whole form again just to get added to a new allocation?!  I don't see another way but that's a huge pain!

Reply:
Okay, so I'll move this to cli115 as soon as I have permission.